src/features/pass-survey/model/hooks/useActivityStepper.ts

```diff
@@ -15,10 +15,18 @@ function useActivityStepper(state: ActivityState | undefined) {
   const step = state?.step ?? 0;
   const items = state?.items ?? [];
   const answers = state?.answers ?? {};
+  const actions = state?.actions ?? [];
 
+  const lastAction = actions[actions.length - 1];
   const currentPipelineItem = state?.items[step];
+  const nextPipelineItem = state?.items[step + 1];
+
+  const shouldSkipNextUserAction =
```

Comment:
Please rename to "shouldIgnoreUserActionTrack" as we've already Skip action and this variable name confusing